module-2/backend/week_three.md

```diff
@@ -10,14 +10,47 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+rails new *program_name* -T --database=postgresql --skip-spring --skip-turbolinks
+
 2. What do Models generally inherit from in rails?
+ApplicationRecord
+
 3. What do Controllers generally inherit from in a rails project?
+ApplicationController
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+resources :horses, only: :index
+(or put multiple routes in an array)
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+rake routes - gives you the prefix(path), the url, and controller#action
+
 6. What is an example of a route helper? When would you use them?
+new_horse_path - use it in a redirect_to
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+a url is the url path in the web address, and the path is the helper used within rails for redirects/ link tos, etc
+
 8. What are strong params and why are the necessary?
+params.require(:blahblah).permit(:params, :params)
+privatize the method AND reduce redundancy
```

Comment:
Also cleans our params so that users can't change attributes we don't want them to change.